documentation/modules/upgrading/proc-downgrade-kafka-kraft.adoc

```diff
@@ -0,0 +1,95 @@
+// This module is included in the following assemblies:
+//
+// assembly-downgrade.adoc
+
+[id='proc-downgrade-kafka-kraft-{context}']
+= Downgrading KRaft-based Kafka clusters and client applications
+
+[role="_abstract"]
+Downgrade a KRaft-based Strimzi Kafka cluster to an earlier version.
+When downgrading a KRaft-based Strimzi Kafka cluster to a lower version, like moving from {KafkaVersionHigher} to {KafkaVersionLower}, ensure that the metadata version used by the Kafka cluster is a version supported by the Kafka version you want to downgrade to. 
+The metadata version for the Kafka version you are downgrading from must not be higher than the version you are downgrading to.
+
+NOTE: Consult the Apache Kafka documentation for information regarding the support and limitations associated with KRaft-based downgrades.
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
+* Before you downgrade the Strimzi Kafka cluster, check the following for the `Kafka` resource:
+
+** The `Kafka` custom resource does not contain options that are not supported by the Kafka version being downgraded to.
+** `spec.kafka.metadataVersion` is set to a version that is supported by the Kafka version being downgraded to.   
+
+.Procedure
+
+. Update the Kafka cluster configuration.
++
+[source,shell,subs=+quotes]
+kubectl edit kafka <kafka_configuration_file>
+
+. Change the `metadataVersion` version to a version supported by the Kafka version you are downgrading to; leave the `Kafka.spec.kafka.version` unchanged at the _current_ Kafka version.
++
+For example, if downgrading from Kafka {KafkaVersionHigher} to {KafkaVersionLower}:
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    metadataVersion: {KafkaMetadataVersionLower} # <1>
+    version: {KafkaVersionHigher} # <2>
+    # ...
+----
++
+<1> Metadata version is changed to a version supported by the earlier Kafka version.
+<2> Kafka version is unchanged.
++
+NOTE: The value of `metadataVersion` must be a string to prevent it from being interpreted as a floating point number.
+
+. Save the change, then wait for rolling updates to complete.
```

Comment:
This will not have rolling update. But the `.status.kafkaMetadataVersion` would be updated by the operator once/if the metadata version is changed.

Reply:
Updated.
I also updated the section on _Checking the status of an upgrade_ so that we now show kafkaMetadataVersion too.